app/scripts/modules/google/loadBalancer/loadBalancer.setTransformer.ts

```diff
@@ -24,21 +27,21 @@ export class GceLoadBalancerSetTransformer {
   }
 
   static get $inject() {
-    return ['elSevenUtils'];
+    return ['gceHttpLoadBalancerUtils'];
   }
 
-  constructor(private elSevenUtils: any) {}
+  constructor(private gceHttpLoadBalancerUtils: GceHttpLoadBalancerUtils) {}
 
-  public normalizeLoadBalancerSet = (loadBalancers: any[]): any[] => {
-    let [elSevenLoadBalancers, otherLoadBalancers] = partition(loadBalancers, this.elSevenUtils.isElSeven);
+  public normalizeLoadBalancerSet = (loadBalancers: IGceLoadBalancer[]): IGceLoadBalancer[] => {
+    let [httpLoadBalancers, otherLoadBalancers] = partition(loadBalancers, (lb) => this.gceHttpLoadBalancerUtils.isHttpLoadBalancer(lb));
```

Comment:
It retains the `this` binding for the method, which doesn't matter here (since the method doesn't depend on `this`), but its probably a better practice to do this consistently.